examples/tikhonov_regularization/example_tik_multi_lam.cpp

```diff
@@ -1,3 +1,21 @@
+//
+// Example on how to solve a Tikhonov regularized least squares problem with
+// multiple λ's using Eldén's bidiagonalization algorithm. The problem is
+// first "pushed" to bidiagonal form. Then a λ is tried. After this we have
+// access to ||x||₂ and ||b-Ax||₂ and can decide on a new λ. And repeat
+// until a suitable λ is found. Once a suitable λ is found, a last
+// transformation is done on x to "pop" the solution to the original problem
+// (the one not in bidiagonal form).
+//
+// We use Eldén's Tikhonov regularized least squares problem for bidiagonal
+// matrix algorithm.
+//
+// Other methods would be
+// (*) Normal Equations (and Cholesky) which is unstable;
```

Comment:
change "unstable" to "not as stable" and also add "and it is not that obvious how to update a Cholesky factorization when the diagonal is changed"